pallets/associated-accounts/src/lib.rs

```diff
@@ -75,16 +75,16 @@ pub mod pallet {
     #[pallet::event]
     #[pallet::generate_deposit(pub(super) fn deposit_event)]
     pub enum Event<T: Config> {
-        /// Altvr username updated  \[owner, username\]
-        AltVRUsernameChanged(T::AccountId, ternoa::String),
+        /// Altvr username created  \[owner, username\]
+        AltVRUsernameCreated(T::AccountId, ternoa::String),
     }
 
     #[pallet::error]
     pub enum Error<T> {
-        /// Altvr username is too short.
-        TooShortUsername,
-        /// Altvr username is too long.
-        TooLongUsername,
+        /// Altvr username length is short (less than what is defined in MinNameLength).
+        ShortUsernameLength,
+        /// Altvr username length is long (more than what is defined in MaxNameLength).
+        LongUsernameLength,
```

Comment:
When your tx has failed and you checked the error and it says "TooLongUsername" you can without doubt say that the reason why it has failed is because your username was too long. LongUsernameLength doesn't tell you that your username was too long.

Example: `TooManyNominators` and `TooManyValidators` are errors that the standard staking pallet has and in that sense it would be OK for us to have `TooShortUsername` or `TooLongUsername` errors.

Reply:
Don't forget that all the comments and names are visible in the Polkadot UI so it needs to make sense from the readers perspective and not from the code perspectives.

Reply:
Since most of the changes occurred on the errors By removing the Too prefix can you tell the changes made and that are ok to keep